coreAPI/src/main/java/net/java/games/input/DefaultControllerEnvironment.java

```diff
@@ -153,6 +112,55 @@ public DefaultControllerEnvironment() {
         }
         return ret;
     }
+
+	/**
+	 * This method will populate the controllers list and is meant to be called manually
+	 * in order to refresh the list of available controllers.
+	 */
+	public void getNewControllers(){
```

Comment:
This method doesn't only get the new controllers, please consider renaming the method indicative of it's function.